### DISCUSSION TITLE
jinja2 partial match?

### DISCUSSION THREAD

**Discussion by mostdaysarebaddays** [netbox-community/netbox#15604] [Help Wanted!] [answered]
The filters work fine for me via the standard API but the config renderer/jinja2 doesn't seem to work with the filtering.

Is there a known method instead for partial match on lookups?

Query:

{%- for int in device.interfaces.filter(enabled="True",mode="tagged",type="10gbase-x-sfpp",description__ic="test:") %}

Response:

<class 'django.core.exceptions.FieldError'>

Unsupported lookup 'ic' for CharField or join on the field not permitted.

Python version: 3.8.12
NetBox version: 3.6.4

**Comment by jeremystretch** [comment, marked as answer]
The API filters are different from the underlying queryset field lookups. E.g. case-insensitive "contains" is `icontains`. Reference the [Django documentation](https://docs.djangoproject.com/en/5.0/ref/models/querysets/#field-lookups) for the complete list.
